common/views/components/LabelsList/LabelsList.js

```diff
@@ -7,15 +7,14 @@ type Props = {|
   labels: {
     url: ?string,
     text: string
-  }[],
-  isSpaced?: boolean
+  }[]
 |}
 
-const LabelsList = ({labels, isSpaced = false}: Props) => (
+const LabelsList = ({labels}: Props) => (
   <ul className={`flex-inline plain-list no-margin ${spacing({s: 0}, {padding: ['top', 'bottom', 'left']})} ${spacing({s: 2}, {padding: ['right']})}`} style={{flexWrap: 'wrap'}}>
     {
       (labels.filter(Boolean).map((label, i) => (
-        <li key={`${label.text}-${i}`} style={{float: 'left', marginRight: isSpaced ? sized(1) : '1px', marginTop: isSpaced ? sized(1) : '1px'}}>
+        <li key={`${label.text}-${i}`} style={{float: 'left', marginRight: sized(0.5), marginTop: sized(0.5)}}>
```

Comment:
Hmm. 
Should we be thinking about setting the base to 3?

Reply:
I wondered, but fairly quickly came down on the side of, 'no, we shouldn't'.

At a certain point (I think I'd argue at any point lower than we currently have, i.e. 6px) the spacing unit would cease to be meaningfully useful because so many potential spacings are possible (you'd almost be better off inlining pixel values).

While I think there might be a handful of instances where 3px is a unit things might be spaced by, I don't think we should go on to allow all multiples of 3. We'd need twice as many spacing classes as we currently have, but (perhaps more importantly) it also increases the granularity, and therefore the complexity, of the decisions we have to make regarding how much spacing should be applied to things.

I actually thought twice about using `spacing()` here at all – perhaps `3px` would be better?

I would _almost_ prefer to suggest a sort of exponential spacing system along the lines of

1px
3px
6px
12px
24px
48px
96px

My two cents, anyhow.

Reply:
A slightly less extreme version of what I'm suggesting, I suppose: https://design-system.service.gov.uk/styles/spacing/

Reply:
Either way, I'ma merge this and we can thrash out the details later!

Reply:
Yea, having these base units are made to be able to create new abstractions from them, similar to s, m, l, xl.

If we want to do this with spacing, and it makes things more consistent, let's!